src/App.tsx

```diff
@@ -1,24 +1,55 @@
-import React from 'react';
 import logo from './logo.svg';
+import axios from 'axios';
 import './App.css';
+import { usePAPforms } from './usePAPforms';
+import { UserForm } from './UserForm';
+import { useState, FormEvent } from 'react';
+import { FormData } from './types';
+
+const INITIAL_DATA: FormData = {
+    filePath: '',
+    password: '',
+};
 
 function App() {
+    const [data, setData] = useState(INITIAL_DATA);
```

Comment:
let's implement get requests to display the server's current settings as the initial data